Modules/System/SharePoint/auth/IAuthorization.Interface.al

```diff
@@ -0,0 +1,4 @@
+interface "SP IAuthorization"
```

Comment:
I can't find any implementation codeunit.

Reply:
Hi, We were thinking about implementation codeunit to add it to pull request but that depends on the Authorization that someone want to use and most likely it will not be part of the System but somone else solution. We developed one for us that use OAuth2 and we can share it as well but maybe not part of the pull request.

Of course if MS would not decide differently